README.md

```diff
@@ -14,18 +14,20 @@ Behind the scenes, `basic-webserver` uses Rust's high-performance [hyper](https:
 
 ## Example
 
-Hello world webserver:
-
-```elixir
-app "helloweb"
-    packages { pf: "https://github.com/roc-lang/basic-webserver/releases/download/0.4.0/iAiYpbs5zdVB75golcg_YMtgexN3e2fwhsYPLPCeGzk.tar.br" }
-    imports [
-        pf.Stdout,
-        pf.Task.{ Task },
-        pf.Http.{ Request, Response },
-        pf.Utc,
-    ]
-    provides [main] to pf
+To run any of the examples, you will need to get [the latest release URL](https://github.com/roc-lang/basic-webserver/releases) which is named using a hash of the file contents, e.g. "Vq-iXfrRf-aHxhJpAh71uoVUlC-rsWvmjzTYOJKhu4M.tar.br". You can usually right-click in your browser to copy the link URL.
+
+Then you need to replace the local path to the platform `"../platform/main.roc"` with the URL you copied in the `app` block of the Roc file.
+
+```roc
+app [main] {
+    # replaced the pf: "../platform/main.roc" with the following
+     pf: "https://github.com/roc-lang/basic-webserver/releases/<latest release hash>.tar.br"
```

Comment:
This should be `pf: platform "..."`, right?

Reply:
Done